sale_order_customer_free_ref/models/sale_order.py

```diff
@@ -0,0 +1,31 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    client_order_ref = fields.Char(
+        compute="_compute_client_order_ref",
+        string="Customer Reference",
+        store=True,
+        copy=False,
+        readonly=True,
+    )
+    customer_order_number = fields.Char(string="Customer Order Number", copy=False)
+    customer_order_free_ref = fields.Char(string="Customer Free Reference", copy=False)
```

Comment:
Shouldn't this be the other way?, i.e. to make ```customer_order_number``` and ```customer_order_free_ref``` computed depending on the value set in the standard field ```client_order_ref```?

Reply:
Not what was requested, but I can hear that `client_order_ref` being readonly now could be problematic.